stress/src/logs.rs

```diff
@@ -88,7 +88,7 @@ fn main() {
 
 fn test_log() {
     error!(
-        name = "CheckoutFailed",
+        name : "CheckoutFailed",
```

Comment:
in case anyone wondering what this is doing - previous "name" was just any attribute, so the string has to copied to new heap memory.
now, name is part of metadata of `tracing`, and is `static. Otel stores it as event_name, which is `static str, so allocation and copying is gone.